docs/api.rst

```diff
@@ -170,8 +169,8 @@ Example Response
 ::
 
     {
```

Comment:
Just realized these two problems:

- This line is not a valid json, there is no need for the additional curly bracket.
- JSON only uses double quotes for strings, so all strings should use `"` instead of `'`.

Could you please fix this?